types/chai-webdriverio/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Don't disable all of this linting rules unless there's a very good reason. Please only use the extends.